addons/web/static/tests/setup.js

```diff
@@ -246,13 +246,23 @@ function removeUnwantedAttrsFromTemplates(attrs) {
     function replaceAttr(attrName, prefix, element) {
         const attrKey = `${prefix}${attrName}`;
         const attrValue = element.getAttribute(attrKey);
-        element.removeAttribute(attrKey);
+        if (element.nodeName === "img") {
+            if (attrKey === "src") {
+                element.setAttribute(attrKey, "/web/image/web.image_placeholder");
+            } else if (attrKey === "t-att-src" && attrKey === "t-attf-src") {
+                element.setAttribute(attrKey, "'/web/image/web.image_placeholder'");
+            }
```

Comment:
```suggestion
            if (["src", "t-att-src", "t-attf-src"].includes(attrKey)) {
                element.setAttribute(attrKey, "/web/image/web.image_placeholder");
            } 
```
What about this?

Reply:
The src case is different than the t-att ones :)

Reply:
Hoooo, sorry, quotes are a pain to read :facepalm: